book-service-rest/src/main/java/com/example/Library/exceptions/IncorrectGenderException.java

```diff
@@ -0,0 +1,7 @@
+package com.example.Library.exceptions;
+
+public class IncorrectGenderException extends RuntimeException{
```

Comment:
It seems like a common validation exception, many individual validation exceptions could be replaced with one common. (so only validation massage will be different)